logger/src/logger.cpp

```diff
@@ -56,4 +57,36 @@ void logMessage(const std::string& message,
     logMessage(Loglevel::INFO, message, location);
 }
 
+void fetchNLogs(size_t noOfLogs)
+{
+    try
+    {
+        std::fstream logfile("/tmp/logdata.txt",
+                             std::ios::trunc | std::ios::out);
+        if (!logfile.is_open())
+        {
+            throw std::runtime_error("Error opening the file!");
```

Comment:
append file name as well, so will come to know which file failed to open ..

Reply:
done